src/tools/sssctl/sssctl_cache.c

```diff
@@ -295,10 +295,12 @@ static const char *sssctl_create_filter(TALLOC_CTX *mem_ctx,
     const char *filter;
     char *filter_value;
     bool qualify_attr = false;
+    bool use_alias = false;
 
     if (strcmp(attr_name, SYSDB_NAME) == 0) {
         if (obj_type == CACHED_USER || obj_type == CACHED_GROUP) {
             qualify_attr = true;
+            use_alias = true;
```

Comment:
I think you are right. It will not harm if we use alias all the time, but currently we only use them users and groups. I will change it to always use the filter with an alias.